weave_demo.ipynb

```diff
@@ -0,0 +1,1199 @@
+       " 'integrity_data': [{'file_size': 46,\n",
+       "   'hash': 'd7c3ccaccf38fb503bff57510e470eae1501be661e4a97d3a2007686cf1f9d40',\n",
+       "   'access_date': '2023-10-19T19:49:54.671651+00:00',\n",
+       "   'source_path': 'WeaveDemoText.txt',\n",
+       "   'byte_count': 100000000,\n",
+       "   'stub': False,\n",
+       "   'upload_path': 'weave-demo-pantry/test-1/acc530466eb811eeb8b60242ac1f000a/WeaveDemoText.txt'}]}"
+      ]
+     },
+     "execution_count": 11,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "basket.get_supplement()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c9afa0d0-42ea-4277-86f5-58418a499c1d",
+   "metadata": {},
+   "source": [
+    "Now let's access the metadata for our basket. Metadata is data the user may add when uploading a basket to a pantry."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 12,
+   "id": "95ffc12c-6053-43de-a47e-1a00b273cce4",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{'Data Type': 'text'}"
+      ]
+     },
+     "execution_count": 12,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "basket.get_metadata()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9338c1a2-a19c-4ff0-a151-d320b5aa76e7",
+   "metadata": {},
+   "source": [
+    "Much like the linux ls command, weave's ls lists files and directories within the file system."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 13,
+   "id": "286b1cb7-6abc-4707-bc30-a163aec301a2",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "['/home/jovyan/work/weave/weave-demo-pantry/test-1/acc530466eb811eeb8b60242ac1f000a/WeaveDemoText.txt']"
+      ]
+     },
+     "execution_count": 13,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "basket_contents = basket.ls()\n",
+    "basket_contents"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ebde9549-4646-4964-87d7-389b874f8982",
+   "metadata": {},
+   "source": [
+    "## Data Provenance\n",
+    "\n",
+    "Weave handles much of its data provenance tracking through the creation of baskets. A basket is meant to represent an atomic data product. It can contain whatever a user wishes to put in the basket, but it's intended purpose is to hold a single instance of one type of data, be it an image, video, text file, or curated training set. A basket in its entirety contains the actual data files specified by the user along with the supplemental files that Weave creates. These supplemental files contain data integrity information, arbitrary metadata specified by the user, and lineage artifacts. Baskets are created at their time of upload and uploaded in an organized state to the data store. Parent UUIDS are the UUIDS Basket(s) that created the current basket."
```

Comment:
I think most of this should go at the start of the basket section. The top of the section only says "here we'll talk about baskets", but then learn about them towards the end of the section. This description may be nice higher up.